webapp/ordering/cron.py

```diff
@@ -170,3 +170,21 @@ def do(self):
         if order_round.is_over and order_round.rides_mails_sent is False:
             print("Sending ride mails!")
             order_round.send_ride_mails()
+
+
+class SendPrepareRideMails(CronJobBase):
+    RUN_EVERY_MINS = 30
+
+    schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
+    code = 'ordering.send_prepare_ride_mails'
+
+    def do(self):
+        print("SendPrepareRideMails")
+        order_round = get_current_order_round()
+        print("Order round: %s" % order_round)
+        if order_round.is_open:
+            next_order_round = get_next_order_round()
+            print("Next order round: %s" % next_order_round)
+            if next_order_round.prepare_ride_mails_sent is False:
```

Comment:
I think this will throw an exception if there's no next order round.

Reply:
True... But the same will happen when there's no current order round? I've added a check though.

Reply:
I think `get_current_order_round` is designed to always return an order round, with one exception: if there are no order rounds.

Reply:
Ah alright and that's only an issue when you start a completely clean version, which is fine. 

I've added a check. Could you review again?